src/pgduckdb_hooks.cpp

```diff
@@ -52,6 +52,23 @@ IsCatalogTable(List *tables) {
 	return false;
 }
 
+static bool
+IsPartitionedTable(List *tables) {
```

Comment:
I think `Is` is the wrong prefix here and the argument name `tables` is also confusing. We pass it a list of tables RTEs not a single table. I realize the problem is the case for `IsCatalogTable`, but we should change this there too (not in this PR though).
```suggestion
ContainsPartitionedTable(List *rtes) {
```

Reply:
Updated the IsCatalogTable function information #417